MauiAppWithBroker/MSALClient/MSGraphHelper.cs

```diff
@@ -93,6 +93,16 @@ public async Task<Stream> GetMyPhotoAsync()
             return userPhoto;
         }
 
+        /// <summary>
+        /// Resets the GraphClientService used by this class
+        /// </summary>
+        /// <returns></returns>
+        public void ResetGraphClientService()
+        {
+            _graphServiceClient = null;
+        }
+
```

Comment:
nit: extra line

Reply:
I cannot add comment in #139, HttpClient should be disposed. Also, this method need not be async.